spec.html

```diff
@@ -125,6 +125,61 @@ <h1>Updated Productions</h1>
   </emu-clause>
 </emu-clause>
 
+<emu-clause id="sec-new-ecmascript-specification-types">
+  <h1>New ECMAScript Specification Types</h1>
+
+  <emu-clause id="sec-classes-specification-types">
+    <h1>Classes Specification Types</h1>
+
+    <emu-clause id="sec-elementdescriptor-specification-type" aoid=ElementDescriptor>
+      <h1>The ElementDescriptor Specification Type</h1>
+
+      <p>
+        The ElementDescriptor is a Record used to represent class elements at runtime.
```

Comment:
The first mention of a specification type should be wrapped in `<dfn></dfn>`, e.g. `<dfn>Element Descriptor</dfn>`. This helps with ECMArkup's auto-linking behavior.